substrate/frame/alliance/src/lib.rs

```diff
@@ -135,9 +134,14 @@ type NegativeImbalanceOf<T, I> = <<T as Config<I>>::Currency as Currency<
 
 /// Interface required for identity verification.
 pub trait IdentityVerifier<AccountId> {
+	type FieldsBitFlags;
+
+	/// Returns the relevant identities that an account is required to have set.
+	fn required_identities() -> Self::FieldsBitFlags;
+
 	/// Function that returns whether an account has an identity registered with the identity
 	/// provider.
-	fn has_identity(who: &AccountId, fields: u64) -> bool;
+	fn has_identity(who: &AccountId, fields: Self::FieldsBitFlags) -> bool;
```

Comment:
These 2 functions could be collapsed into `has_required_identities(who: &AccountId, fields: Self::FieldsBitFlags) -> bool` but I wasn't sure it's better than explicitly declaring the relevant fields in a standalone function as it is now. Let me know what you think.

Reply:
Ha nice! I had the same idea :D And I think we should do it this way. No need to have these two functions!

Reply:
Done.